packages/connect-node/src/defer.ts

```diff
@@ -0,0 +1,25 @@
+export function defer<T>(): Promise<T> & Ctrl<T> {
+  let res: ((v: T | PromiseLike<T>) => void) | undefined = undefined;
+  let rej: ((reason?: unknown) => void) | undefined;
+  const p = new Promise<T>((resolve, reject) => {
+    res = resolve;
+    rej = reject;
+  });
+  void p.catch(() => {
+    //
```

Comment:
```
    // We want to provide several promises that are typically awaited
    // by the user one after the other.
    // If we reject one of the promises before it is awaited by the user,
    // some runtimes may report an unhandled promise rejection.
    // We are attaching this error handler to avoid the warnings.
```

Added this as a comment.